data/chameleoncloud/sites/uc/clusters/chameleon/nodes/219a58dd-a3b2-4c4f-8517-ab72bc82c741.json

```diff
@@ -5,111 +5,96 @@
     "smt_size": 48
   },
   "bios": {
-    "release_date": "05/21/2018",
-    "vendor": "Dell Inc.",
-    "version": 1.4
+    "release_date": "2024-02-19",
+    "vendor": "Dell",
+    "version": "2.21.2"
   },
   "chassis": {
-    "manufacturer": "Dell Inc.",
+    "manufacturer": "Dell",
     "name": "PowerEdge R740",
     "serial": "9YTY7M2"
   },
   "gpu": {
     "gpu": true,
     "gpu_count": 1,
-    "gpu_model": "RTX 6000",
+    "gpu_model": "TU102GL [Quadro RTX 6000/8000]",
```

Comment:
another place we should be more specific, I believe there's a pci sub-identifier which can distinguish which tu102 model it is.